gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -171,6 +203,26 @@ components:
             from this endpoint invocation. If not provided, it will default to "complete".
           type: string
           nullable: true
+    UserTaskAssignmentRequest:
+      type: object
+      properties:
+        assignee:
+          description: The assignee for the user task. The assignee must not be empty or null.
+          type: string
+          nullable: false
+        allowOverride:
+          description: >
+            By default, the task is reassigned if it was already assigned. If this option is set
+            to false then an error will be returned in such cases. The task must then first be
+            unassigned in order to be assigned again.
```

Comment:
```suggestion
            By default, the task is reassigned if it was already assigned. Set this to `false`
            to return an error in such cases. The task must then first be unassigned to
            be assigned again. Use this when you have users picking from group task
            queues to prevent race conditions.
```